### PR TITLE
IGAPP-444: Create intelliJ run configs

### PR DESCRIPTION
**Pull request by SteffiStuffel**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

**Review by klinzo — APPROVED** [review]

File: web/.idea/runConfigurations/Run__aschaffenburg_.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<component name="ProjectRunConfigurationManager">
```
Comment (maxammann) [inline web/.idea/runConfigurations/Run__aschaffenburg_.xml:1]: I think this duplicates `Start_packager__aschaffenburg_.xml`

File: web/.idea/runConfigurations/Run__malte_.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="Run (malte)" type="js.build_tools.npm">
```
Comment (maxammann) [inline web/.idea/runConfigurations/Run__malte_.xml:2]: Should be `Start packager (malte)` If I remember correctly.

**Review by maxammann — APPROVED** [review]

File: web/.idea/runConfigurations/Run__aschaffenburg_.xml (hunk 1 above)
Comment (klinzo) [reply web/.idea/runConfigurations/Run__aschaffenburg_.xml:1]: I thought the same but this is for web, so no duplicates here ^^

File: web/.idea/runConfigurations/Run__malte_.xml (hunk 2 above)
Comment (klinzo) [reply web/.idea/runConfigurations/Run__malte_.xml:2]: This is for web so I guess it is fine.

File: web/.idea/runConfigurations/Run__aschaffenburg_.xml (hunk 1 above)
Comment (maxammann) [reply web/.idea/runConfigurations/Run__aschaffenburg_.xml:1]: upsi

File: web/.idea/runConfigurations/Run__malte_.xml (hunk 2 above)
Comment (maxammann) [reply web/.idea/runConfigurations/Run__malte_.xml:2]: upsi

**Pull request merged**